BUILDBOT.md

```diff
@@ -0,0 +1,134 @@
+# Connecting a WebKit Buildbot
+The `docker-webkit-dev` project contains Docker images for running Buildbot
+workers that connect to WebKit infrastructure. The Windows WebKit port builds
+happen within Docker containers running the `webkitdev/buildbot-worker` image.
+
+> [!IMPORTANT]  
+> To connect to WebKit infrastructure credentials are required.
+> Contact the infrastructure team on [Slack](https://webkit.slack.com) for
```

Comment:
Is there a specific group or anything for this? People might be a problem with changing, but it might be hard for someone to know who to ask for just infrastructure team.